### DISCUSSION TITLE
NuGet - packages.lock.json - No tool releases found.

### DISCUSSION THREAD

**Discussion by maartenvandiemen** [renovatebot/renovate#20429] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

packages.lock.json automatically updated when a new dependency is available. This should be supported as described here: https://docs.renovatebot.com/nuget/

### What you tried so far.

Pull request on which this issue occurs: https://github.com/mvdiemen/TodoItems/pull/27

Searched in relevant logs, which states the same message as in the PullRequest.
Added nuget.config to the repo, thought this might be helpfull.

**Comment by rarkins** [comment, marked as answer]
Can you tick the rebase/retry checkbox in the PR?

**Reply by maartenvandiemen** [reply]
Even though I tried this multiple times during the afternoon. It now suddenly works. Thanks! (even though I have no clue what I changed/has been changed)

**Reply by rarkins** [reply]
We had a temporary bug which was then fixed, sorry for the interruption

**Reply by viceice** [reply]
fixed in
- https://github.com/renovatebot/renovate/pull/20415
